pkg/cincinnati/cincinnati.go

```diff
@@ -188,3 +289,22 @@ func (e *edge) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+func convertRetrievedUpdateToRelease(update node) (configv1.Release, error) {
+	cvoUpdate := configv1.Release{
+		Version: update.Version.String(),
+		Image:   update.Image,
+	}
```

Comment:
There is no check for cases where update.Image is null. It might be ok as the code to validate the version will be called at some point of time before the actual upgrade. However I wanted to point it out.